dev/run

```diff
@@ -24,10 +24,15 @@ import optparse
 import os
 import posixpath
 import re
+import socket
 import subprocess as sp
 import sys
 import time
 import uuid
+import pkgutil
+import imp
```

Comment:
"Deprecated since version 3.4: The imp package is pending deprecation in favor of importlib."

https://docs.python.org/3.5/library/imp.html